src/garage/envs/bullet/bullet_env.py

```diff
@@ -1,19 +1,36 @@
 """Wrappers for py_bullet environments."""
 import inspect
+import math
 
 import akro
 import gym
-from pybullet_envs.bullet.minitaur_duck_gym_env import MinitaurBulletDuckEnv
-from pybullet_envs.bullet.minitaur_gym_env import MinitaurBulletEnv
-from pybullet_envs.env_bases import MJCFBaseBulletEnv
+from gym.wrappers.time_limit import TimeLimit
 
+from garage import Environment, EnvStep, StepType
 from garage.envs.env_spec import EnvSpec
```

Comment:
move `EnvSpec` to `_environment.py` (or create an issue)

Reply:
done